apps/activities/blocks.py

```diff
@@ -0,0 +1,19 @@
+from django.utils.translation import ugettext_lazy as _
+from wagtail.wagtailcore import blocks
+
+from .models import Activity
+
+
+class PlatformActivityBlock(blocks.StructBlock):
+    heading = blocks.CharBlock(label=_('Heading'))
+    count = blocks.IntegerBlock(label=_('Count'), default=5)
+
+    def get_context(self, value, parent_context=None):
+        context = super().get_context(value, parent_context=parent_context)
+        block = context['self']
+        context['activities'] = Activity.objects.all()[:block['count']]
```

Comment:
I am not sure whether additional filters are necessary here, e.g. to remove activities from private projects.